Testing/PerformanceTests/analysis.py

```diff
@@ -302,13 +253,16 @@ def make_environment_html(res):
     """ % (res['host'], res['environment'], res['runner'])
     return html
 
-#============================================================================================
+
+# ============================================================================================
 def make_detailed_html_file(basedir, name, fig1, fig2, last_num):
     """ Create a detailed HTML report for the named test """
-    html = default_html_header
+    html = """<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd"><html>
```

Comment:
We seemed to have defined this header string twice. Can we define this once at the head of the file?